src/Compilers/Server/VBCSCompilerTests/DesktopBuildClientTests.cs

```diff
@@ -18,42 +18,6 @@ namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
     public class DesktopBuildClientTests : TestBase
```

Comment:
I like making it `abstract` but it won't prevent adding tests.  xunit will just run them for every derived type which also has at least one `[Fact]` attribute.